src/control/SConscript

```diff
@@ -208,6 +219,19 @@ def scons():
         daosbin = install_go_bin(dbenv, gosrc, [daos_hdlrs_lib], "daos", "daos")
         AlwaysBuild([daosbin])
 
+        menv = denv.Clone()
+        menv["ENV"]["LD_LIBRARY_PATH"] = menv.subst(
```

Comment:
if I'm reading this right, you are using daos and dmg to generate man pages?

And you need the library path because the libraries are not installed yet?

Reply:
Yes, running daos/dmg out of the build tree in order to generate the manpages. I wound up having to monkey around with things like this in order to get it to work correctly in the RPM build.